src/views/accounts/AccountChangeSecret/Overview/index.vue

```diff
@@ -107,6 +107,10 @@ export default {
     ::v-deep #HomeCard .el-card.no-border {
       height: 100%;
       margin-bottom: unset !important;
+
+      .ListTable .el-pagination {
+        display: block;
+      }
     }
   }
 
```

Comment:
There were differences between the two versions that could be optimized to create more maintainable code. To improve readability, we can remove unnecessary spaces before and after `.` in the CSS file. Additionally, you might add classes directly instead of using dot notation when applying styles to elements.

The main change is adding a space at the end of `::v-deep .`. Otherwise, all else appears fine! Let's keep it consistent with future updates on your project. 

Also for efficiency sake, I suggest removing the empty line in case it isn't causing any issue (though this depends entirely on what's right for your application).